.github/workflows/build.yaml

```diff
@@ -42,7 +46,7 @@ jobs:
           docker build --build-arg copa_version=${{ env.COPA_VERSION }} -t copa-action .
           cd ${{ github.workspace }}/test
```

Comment:
we can remove this line too i think?

Reply:
yup we can remove this, just need to change the bats command to call ./test/test.bats